File: baekjoon/bronze/level1/test_11050/Main.java
package bronze.level1.test_11050;

import java.io.*;
import java.util.stream.Stream;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        BufferedWriter bw = new BufferedWriter(new OutputStreamWriter(System.out));

        int[] input = Stream.of(br.readLine().split(" ")).mapToInt(Integer::parseInt).toArray();
        int result = factorial(input[0]) / (factorial(input[1]) * factorial(input[0] -input[1]));

        bw.write(String.valueOf(result));
        bw.close();
    }

    static int factorial (int n) {
        int result = 1;
        for (int i = 1; i <= n; i++) {
            result *= i;
        }
        return result;
    }
}

File: baekjoon/bronze/level1/test_11050/kotlin/Main.kt
package baekjoon.bronze.level1.test_11050.kotlin

fun main() {
    val arr = readln().split(" ").map { it.toInt() }
    print(factory(arr[0]) / (factory(arr[0] - arr[1]) * factory(arr[1])))
}

fun factory(n: Int): Int {
    var sum = 1

    for (i in 2..n) {
        sum *= i
    }
    return sum
}
